hyppo/ksample/tests/newtests.py

```diff
@@ -0,0 +1,44 @@
+import abc
+import numpy
+from hyppo.ksample import SmoothCFTest, MeanEmbeddingTest
+from unittest import TestCase
+from numpy.random import seed
+
+
+class StatTests:
```

Comment:
This is a very weird way of writing unit tests. Rewrite according to the conventions of other tests in `hyppo.ksample`

Reply:
edited files

Reply:
hChanged